### PR TITLE
Wire _check_autosave on init

### PR DESCRIPTION
**Pull request by vidartf**
Previously, autosave did not work unless the image/video/audio instance was set, as the observers does not trigger on initialization.

### CODE REVIEW

**Comment by vidartf** [comment]
Remaining issue with autosave:
- Set up and record a value, but with autosave set to false. (`ImageRecorder(stream=stream, filename='foo', recording=True)`), where the stream is unchanging.
- Set autosave and recording to `True`.
- `recording` will be subsequently set to `False`, but no image saved as the new snapshot is identical to the old value (preventing both a sync of the value and the change notification).

**Comment by vidartf** [comment]
Possible solution for the issue: Send a custom message 'snapshot' for the image recorder that triggers an autosave check.

File: ipywebrtc/webrtc.py [hunk 1]
```diff
@@ -348,6 +349,12 @@ class ImageRecorder(Recorder):
     _width = Unicode().tag(sync=True)
     _height = Unicode().tag(sync=True)
 
+    def __init__(self, format='png', filename=Undefined, recording=False, autosave=False, **kwargs):
+        super(ImageRecorder, self).__init__(
+            format=format, filename=filename, recording=recording, autosave=autosave, **kwargs)
+        # Set up initial observer on child:
+        self.image.observe(self._check_autosave, 'value')
```
Comment (maartenbreddels) [inline ipywebrtc/webrtc.py]: if 'image' not in kwargs:

**Comment by vidartf** [comment]
Should be OK now!

**Pull request merged**